src/Make/AbstractMakeZip.php

```diff
@@ -39,6 +40,16 @@ abstract class AbstractMakeZip extends AbstractMake implements ZipMakerInterface
 
     protected string $hashAlgorithm = 'sha256';
 
+    /**
+    * Constructor
+    *
+    * @param AbstractEntity $entity Experiments or Database
+    */
+    public function __construct(AbstractEntity $entity, protected bool $includeChangelog)
```

Comment:
Perhaps I should be consistent about adding default values throughout the entire hierarchy, come to think of it.

Reply:
Yeah, add a default value that matches previous behavior so child classes doesn't need to be modified.

Reply:
> Yeah, add a default value that matches previous behavior
Added. 

> so child classes doesn't need to be modified.
As we instantiate the child classes rather than the parent, and the behaviour should depend on parameter values passed to that constructor, we should provide the parameter value in any case, right? I've added defaults so it should match up in any case.